app/controllers/webauthn_credentials_controller.rb

```diff
@@ -13,6 +13,7 @@ def callback
     webauthn_credential = build_webauthn_credential
 
     if webauthn_credential.save
+      Mailer.webauthn_credential_created(webauthn_credential.id).deliver_later
```

Comment:
Thoughts on doing this in an after_create/destroy callback in the model, so emails are sent regardless of the code path used to create/destroy the credential?

Reply:
Good idea! I prefer doing that because as you mentioned, it doesn't depend on the code paths